### PR TITLE
reef: mgr/dashboard: remove cherrypy_backports.py

### PR DESCRIPTION
**Pull request by nizamial09**
backport tracker: https://tracker.ceph.com/issues/68845

---

backport of https://github.com/ceph/ceph/pull/60602
parent tracker: https://tracker.ceph.com/issues/68813

this backport was staged using ceph-backport.sh version 16.0.0.6848
find the latest version at https://github.com/ceph/ceph/blob/main/src/script/ceph-backport.sh

### CODE REVIEW

**Review by afreen23 — APPROVED** [review]

File: src/pybind/mgr/prometheus/module.py [hunk 1]
```diff
@@ -28,21 +28,6 @@
 
 DEFAULT_PORT = 9283
 
-# When the CherryPy server in 3.2.2 (and later) starts it attempts to verify
-# that the ports its listening on are in fact bound. When using the any address
-# "::" it tries both ipv4 and ipv6, and in some environments (e.g. kubernetes)
-# ipv6 isn't yet configured / supported and CherryPy throws an uncaught
-# exception.
-if cherrypy is not None:
-    Version = packaging.version.Version
```
Comment (adk3798) [inline src/pybind/mgr/prometheus/module.py:37]: I guess this is a bit different in reef
```
prometheus/module.py:4:1: F401 'pkg_resources.packaging' imported but unused
1     F401 'pkg_resources.packaging' imported but unused
```

Comment (nizamial09) [reply src/pybind/mgr/prometheus/module.py:37]: ah, ok. thanks for that. fixed it.

**Comment by vshankar** [comment]
> @yuriw shall we merge this? I think there are still some broken teuthology which could be fixed by merging this.

Can we prioritize merging this since upgrade jobs in fs suite fail from reef upgrade?

**Comment by stuartthebruce** [comment]
This fixed a Rocky Linux 9 system running Ceph 18.2.4 that had the following problem after patching to EL9.5 that included `python3-cherrypy-18.6.1-2.el9.noarch` from EPEL 9. So hopefully this patch can show in in a Reef 18.2.5 EL9 RPM sometime soon.
<pre>
[WARN] MGR_MODULE_DEPENDENCY: 2 mgr modules have failed dependencies
       Module 'dashboard' has failed dependency: No module named 'cherrypy.wsgiserver'
       Module 'prometheus' has failed dependency: Invalid version: 'unknown'
</pre>

**Comment by nizamial09** [comment]
from the run that is scheduled on https://tracker.ceph.com/issues/68838, which included this fix as well, I don't see any error related to this issue. So I am going to merge this now.

**Pull request merged**